### ISSUE TITLE
Support Table Bindings (input) from ServiceBus Custom Properties

### ISSUE THREAD

**Issue by JoeBrockhaus** [Azure/azure-webjobs-sdk#798]
I have [a S/O question](http://stackoverflow.com/questions/39110874/) which highlights my current issue. 

If I json-serialize the model representing the message body, and construct the BrokeredMessage from that string, subsequent Table bindings that rely on properties in that message body fail. 

Being able to bind to those BrokeredMessage Custom Properties would solve this problem.

**Comment by MikeStall** [comment]
Is this saying that the binding data for a SB trigger should include the "custom properties"? 
Normally binding data needs to be static. Maybe related to https://github.com/Azure/azure-webjobs-sdk/issues/968

**Comment by mathewc** [comment]
Yeah, making arbitrary/variable custom properties violates a bunch of rules we have now in the system that binding data is static. However, if there are some non-variable properties in the property bag, those could be promoted to binding data (similar to what we do with Queue dequeue count, etc.). Related issue: https://github.com/Azure/azure-webjobs-sdk/issues/1004

**Comment by mathewc** [comment]
We recently added support for binding to BrokeredMessage.Properties as documented here: https://github.com/Azure/azure-webjobs-sdk/wiki/Trigger-Binding-Data. I believe that gets us half way where you want to go, however there is still no way for you to "dot into" the Properties data in your table binding expression (e.g. can't do `{Properties.PK}`).
